app/src/test/java/gov/va/vro/EvidenceSummaryDocumentProcessorTest.java

```diff
@@ -55,7 +58,7 @@ void positiveEvidenceSummaryDocumentProcessor() throws Exception {
 
     // Call generate-pdf endpoint to activate the Evidence Summary Document processor.
     var mapper = new ObjectMapper();
-    InputStream stream = pdfGeneratorInput01.getInputStream();
+    InputStream stream = pdfGeneratorMas.getInputStream();
```

Comment:
okay now i'm confused. Mas doesn't talk to v1 endpoints? i assume this is just a naming issue.

Reply:
yep this was here cause I made a v2 test for this endpoint, which it turns out we didn't need.  Removing this and making it pdfGeneratorInput01 as it originally was.